src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1335,16 +1370,19 @@ void PolicyHandler::OnActivateApp(uint32_t connection_key,
 }
 
 void PolicyHandler::KmsChanged(int kilometers) {
+  POLICY_LIB_CHECK_VOID();
   LOG4CXX_DEBUG(logger_,
                 "PolicyHandler::KmsChanged " << kilometers << " kilometers");
-  POLICY_LIB_CHECK_VOID();
-  policy_manager_->KmsChanged(kilometers);
+  CallPolicyManagerFunction(&PolicyManager::KmsChanged, kilometers);
 }
 
 void PolicyHandler::PTExchangeAtUserRequest(uint32_t correlation_id) {
-  LOG4CXX_TRACE(logger_, "PT exchange at user request");
   POLICY_LIB_CHECK_VOID();
-  std::string update_status = policy_manager_->ForcePTExchangeAtUserRequest();
+  LOG4CXX_DEBUG(logger_, "PT exchange at user request");
+
+  std::string update_status =
```

Comment:
@SNiukalov Const?

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/98dc17dda0378940c016a6b9a16f1e3992ee8e65